src/pal/tools/gen-buildsys-clang.sh

```diff
@@ -22,10 +22,16 @@ if which "clang-$2.$3" > /dev/null 2>&1
     then
         export CC="$(which clang-$2.$3)"
         export CXX="$(which clang++-$2.$3)"
+        if [ `uname` = "FreeBSD" ]; then
+            export LLVM_HOME=${LLVM_HOME:-/usr/local/llvm$2$3}
+        fi
 elif which "clang$2$3" > /dev/null 2>&1
     then
         export CC="$(which clang$2$3)"
         export CXX="$(which clang++$2$3)"
+        if [ `uname` = "FreeBSD" ]; then
+            export LLVM_HOME=${LLVM_HOME:-/usr/local/llvm$2$3}
```

Comment:
Just found that LLVM_HOME later on is used to set the default paths for locating LLDB library and includes. This fix is handy so that you don't need to see LLDB arguments manually as per current FreeBSD build instructions. This also makes sure the LLDB library matches the compiler version chosen, although that may not be necessary.